src/seedu/addressbook/AddressBook.java

```diff
@@ -549,6 +567,20 @@ private static String executeListAllPersonsInAddressBook() {
         return getMessageForPersonsDisplayedSummary(toBeDisplayed);
     }
 
+    
+    /**
+     * Displays all persons in the address book to the user; in alphabetical order.
+     *
+     * @return feedback display message for the operation result
+     */
+    private static String executeListSortedAllPersonsInAddressBook() {
```

Comment:
Good that you follow the consistency in naming the method. Consider the intended behavior here which should require you to clone the arraylist before sorting. Otherwise, the ordering of the original arraylist will be permanently changed.